data/trie/patriciaMerkleTrie.go

```diff
@@ -394,6 +408,10 @@ func (tr *patriciaMerkleTrie) GetDirtyHashes() (data.ModifiedHashes, error) {
 		return nil, err
 	}
 
+	for hash := range dirtyHashes {
```

Comment:
for _, hash := range oldHashes

Reply:
DirtyHashes is a map[string]struct{}, so it is ok like it is.